.github/workflows/create-py-release-manylinux.yaml

```diff
@@ -1,20 +1,56 @@
 name: Create Python release manylinux
 
 on:
-  push:
-    # Sequence of patterns matched against refs/tags
-    tags:
-      - 'py-polars-v*' # Push events to matching py-polars-v*, i.e. py-polars-v1.0, py-polars-v20.15.10
+  - pull_request
+#  push:
+#    # Sequence of patterns matched against refs/tags
+#    tags:
+#      - 'py-polars-v*' # Push events to matching py-polars-v*, i.e. py-polars-v1.0, py-polars-v20.15.10
 jobs:
   build_manylinux:
     name: Create Release manylinux
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: Publish wheel
-        uses: docker://konstin2/maturin:latest
+#      - name: Publish wheel
+#        uses: docker://konstin2/maturin:latest
+#        env:
+#          MATURIN_PASSWORD: ${{ secrets.PYPI_PASS }}
+#        with:
+#          entrypoint: /bin/bash
+#          args: .github/deploy_manylinux.sh
+
+      - name: Install latest Rust nightly
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: nightly-2022-01-01
+          override: true
+      - name: Add target
+        run: |
+          rustup target add aarch64-unknown-linux-gnu
```

Comment:
Since manylinux wheels are built in docker containers, you don't need to install Rust on the host.